packages/release/activate-multi-user.service

```
@@ -0,0 +1,12 @@
+[Unit]
+Description=Isolate multi-user.target
+After=configured.target
+
+[Service]
+Type=oneshot
+ExecStart=systemctl isolate multi-user
```

As with `activate-configured`:
```suggestion
ExecStart=/usr/bin/systemctl isolate multi-user
```